### PR TITLE
Added Prometheus sample

### PR DESCRIPTION
**Pull request by yuriatgoogle**
This sample is to support the documentation currently staged at
https://cloud.devsite.corp.google.com/stackdriver/docs/solutions/slo-monitoring/sli-metrics/overview

### CODE REVIEW

File: monitoring/prometheus/package.json [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    "test": "mocha -- system-test/*.test.js --timeout=5000 --exit"
+  },
+  "author": "Yuri Grinshteyn",
+  "license": "Apache-2.0",
+  "dependencies": {
+    "express": "^4.17.1",
+    "lint": "^0.7.0",
+    "node-stopwatch": "^0.0.1",
+    "prom-client": "^12.0.0",
+    "request": "^2.88.2",
```
Comment (fhinkel) [inline monitoring/prometheus/package.json:17]: Request is deprecated. Can you use `node-fetch` instead please.

**Review by fhinkel — CHANGES_REQUESTED** [review]

File: monitoring/prometheus/package.json (hunk 1 above)
Comment (yuriatgoogle) [reply monitoring/prometheus/package.json]: Request is there by mistake - removed it.

**Review by fhinkel — APPROVED** [review]

**Comment by yuriatgoogle** [comment]
@fhinkel Thank you! Can you please merge? I don't have access to do so...

**Comment by fhinkel** [comment]
Done :)

**Pull request merged**